HerPortal.UnitTests/Builders/UserBuilder.cs

```diff
@@ -53,5 +47,4 @@ public UserBuilder WithEmail(string email)
         user.EmailAddress = email;
         return this;
     }
-}
-
+}
```

Comment:
Is this auto-formatter removing this? A bit surprising

Reply:
yea this is auto formatted